app.vue

```diff
@@ -1,6 +1,20 @@
 <template>
-  <div>
+  <div
+    class="prose max-w-none
```

Comment:
What is the purpose of these `prose` classes?

Reply:
@AndyClifford this indicates to use the Tailwind typography plugin. 
If you remove this locally you'll be able to see the difference easily 👍 
https://tailwindcss.com/docs/typography-plugin